<ui version="4.0" >
 <author>Diego Iastrubni &lt;diegoiast@gmail.com></author>
 <class>EditorCfgUI</class>
 <widget class="QWidget" name="EditorCfgUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>GroupBox</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="cbMakeBackup" >
            <property name="text" >
             <string>Make backups</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowLineNumbers" >
            <property name="text" >
             <string>Show line numbers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbMarkCurrentLine" >
            <property name="text" >
             <string>Mark current line</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbWorkWrap" >
            <property name="text" >
             <string>Word wrap</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QToolButton" name="btnChooseFont" >
           <property name="text" >
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Editor font</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>End of line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboEOL" >
           <item>
            <property name="text" >
             <string>Don't modify</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Unix (CR)</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>DOS/Window (CR/LF)</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Mac (LF)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <resources/>
 <connections/>
</ui>
